specs-go/version.go

```diff
@@ -11,7 +11,7 @@ const (
 	VersionPatch = 0
 
 	// VersionDev indicates development branch. Releases will be empty string.
-	VersionDev = "-rc6-dev"
+	VersionDev = "-dev"
```

Comment:
> Up until now, -dev has meant “working towards”, so 1.0.0-rc6-dev was followed by 1.0.0-rc6.

Oops, that's not right, since this PR is based on a post-rc6 master using 1.0.0-rc6-dev.  I think I was confused by [pre-release markers sorting earlier in SemVer][1].  So this commit is fine (since we don't care about SemVer sorting between releases), and sorry for the noise.

[1]: http://semver.org/spec/v2.0.0.html#spec-item-9